### ISSUE TITLE
[ACCESSIBILITY] Intro video does not have play/pause controls to stop automatic playing 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1569]
**Describe the accessibility issue**
For moving, [blinking](https://www.w3.org/TR/WCAG21/#dfn-blinking), scrolling, or auto-updating information, all of the following are true:

Moving, blinking, scrolling
For any moving, blinking or scrolling information that (1) starts automatically, (2) lasts more than five seconds, and (3) is presented in parallel with other content, there is a mechanism for the user to [pause](https://www.w3.org/TR/WCAG21/#dfn-pause), stop, or hide it unless the movement, blinking, or scrolling is part of an activity where it is [essential](https://www.w3.org/TR/WCAG21/#dfn-essential); and

Auto-updating
For any auto-updating information that (1) starts automatically and (2) is presented in parallel with other content, there is a mechanism for the user to pause, stop, or hide it or to control the frequency of the update unless the auto-updating is part of an activity where it is essential.

[WCAG Success Criterion 2.2.2 Pause, Stop, Hide](https://www.w3.org/TR/WCAG21/#pause-stop-hide)

**To Reproduce
Steps to reproduce the behavior:**

1. Go to freshly installed version of app (my instance was 3.3.15 apk)
2. Go through app orientation screens
3. Upon arriving at third screen video automatically plays with no option to pause

Expected behavior
No videos should auto play, but if they do they should have a keyboard only way to pause/play the video at their own discretion. 

**Screenshots**
![Screenshot_20220420-143220](https://user-images.githubusercontent.com/82892873/164309870-48321cb6-2706-439e-ac28-e228bd8c034d.png)

**Smartphone (please complete the following information):**
Pixel 4a- Android 12

Additional context

**Comment by dobromirdobrev** [comment]
Play/Pause button is added to the video tutorial panels - in Onboarding and in Settings.
The video does not start automatically anymore. 

This is done in version 3.3.20

**Comment by acovher2** [comment]
Verified & Closing. 
Thank you,
Amanda
